Apps/Sandcastle/gallery/Terrain.html

```diff
@@ -34,16 +34,19 @@
 //Sandcastle_Begin
 var viewer = new Cesium.Viewer('cesiumContainer');
 
-var scene = viewer.scene;
-var globe = scene.globe;
-globe.depthTestAgainstTerrain = true;
-
 var cesiumTerrainProviderMeshes = new Cesium.CesiumTerrainProvider({
     url : '//assets.agi.com/stk-terrain/world',
     requestWaterMask : true,
     requestVertexNormals : true
 });
+viewer.terrainProvider = cesiumTerrainProviderMeshes;
+
+var mountEverest = new Cesium.Cartesian3(300770.50872389384, 5634912.131394585, 2978152.2865545116);
```

Comment:
Yep, agreed that duplicating view code is ugly.  Wrapping it in a function didn't feel right either in a sandcastle example, nor did adding an entry for viewing the entire globe (there's already a home button for that).  My only justification was that this is a sandcastle example, that is exposing code snippets for developer to experimentation, without having to wade through custom setup code.

Long story short, I'll stop down and we can talk about how this can be fixed in sandcastle.